internal/proto/users/v1/users.proto

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Abstract Machines
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package users.v1;
+option go_package = "github.com/absmach/supermq/api/grpc/users/v1";
+
+// UsersService is a service that provides user-related functionalities
+// for SuperMQ services.
+service UsersService {
+  rpc SendEmailWithUserId(SendEmailWithUserIdReq) returns (SendEmailRes) {}
+}
+
+message SendEmailWithUserIdReq {
+  repeated string users = 1;    // Email recipients
+  string from = 2;              // Sender email address (optional)
+  string subject = 3;           // Email subject
+  string header = 4;            // Email header text
+  string user = 5;              // User name
+  string content = 6;           // Email content/URL
+  string footer = 7;            // Email footer text
```

Comment:
Let's make it something like:
```
enum ContactType {
  ID = 0;
  Email = 1;
}

message EmailReq {
  string from = 1;              // Sender email address (optional)
  ContactType fromType = 2;
  repeated string to = 3;       // Email recipients
  ContactType toType = 4;
  string subject = 5;           // Email subject
  string content = 6;           // Email content/URL          // User name
  optional string template = 10;
  map<string, string> options = 11;
}
```
We will see what to put instead of this map as we progress with the server implementation.

Reply:
In the future, we can be able to send body already prepared (template parsed), but also we should be able to send something like `template_file` name, so that emailer takes care of it.